website/templates/website/ourwork.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'website/base.html' %}
+{% block content %}
+<img src="/media/images/ourwork.jpg" style="width:100%">
```

Comment:
leave a space after the colon:

```suggestion
<img src="/media/images/ourwork.jpg" style="width: 100%">
```

There are more like this so please fix them all